Dockerfile-frontend

```diff
@@ -27,7 +30,8 @@ RUN echo "export const environment = {production: false, REST_API_URL:  'http://
 # Use the DIST folder to package an image with NGINX.
 ####################################################################
 
-FROM nginx:1.19.4 AS runtime
+FROM $DOCKER_HUB/library/nginx:$NGINX_VERSION AS runtime
+
```

Comment:
I made this change upstream as well https://github.com/spring-petclinic/spring-petclinic-reactive/pull/15